### ISSUE TITLE
Implement react redux

### ISSUE THREAD

**Issue by omrajsharma** [omrajsharma/bigohhh.com#44]

**Comment by muskandn** [comment]
Hello @omrajsharma 
I want to contribute to this issue under GSSoC'24.
Please consider me assigning it to me.

**Comment by 0xJaskeerat** [comment]
Hey @omrajsharma , I would suggest to use Redux Toolkit , itsss wayy better than using plain redux 
React-redux setup can be done with that only , leme know if you think can be a better idea
If yeah , let's discuss on this and then you can assign me to work on this

**Comment by muskandn** [comment]
Yes even I was talking about using Redux Toolkit only as officially it has been instructed to avoid Redux and to use Redux Toolkit. @omrajsharma Please consider me assigning it to me.

**Comment by omrajsharma** [comment]
@muskandn I'm assigning this issue to you. if you need any help then do let me know.

**Comment by muskandn** [comment]
> @muskandn I'm assigning this issue to you. if you need any help then do let me know.

Yeah sure... I will start working on it asap
